src/main/java/sggw/wzim/czasnawypad/controller/AttractionFavouriteController.java

```diff
@@ -32,5 +32,10 @@ ResponseEntity<?> addFavourite(@RequestBody @Valid CreateFavouriteAttractionDTO
         return ResponseEntity.ok().body(result);
     }
 
+    @DeleteMapping("/{id}")
+    ResponseEntity<?> deleteFavourite(@PathVariable Integer id) {

```

Comment:
ResponseEntity Void would be correct here, as well as more verbose variable name